runtime/gc_structs/VMThreadStackSlotIterator.hpp

```diff
@@ -43,6 +43,14 @@ typedef void J9MODRON_OSLOTITERATOR(J9JavaVM *javaVM, J9Object **objectIndirect,
  */
 class GC_VMThreadStackSlotIterator
 {
+private:
+	static void initializeStackWalkStateForGC(
```

Comment:
Not sure how this compiles though, since at the spot the method is defined in cpp file, it does not seem like the method is a member of the class (what made say it needs more specific scope)!?

Reply:
ah, I understand now.... in *cpp this method is defined before their users, so declaration was not needed.
this declaration is unused
but it's probably the best approach: leave this declaration, remove 'forGC' (since the class gives specific scope) and fix cpp file so that method is part of this class